api/src/database/migrations/scopes.migration.ts

```diff
@@ -0,0 +1,9 @@
+import type { Migration } from "../config/databaseClient";
```

Comment:
In practice, once released, this is the proper way to update via the migrations.

I think due to this not being in production yet, we can get away with keeping this in the initial migration script.

Reply:
Was throwing an Error locally due to having a column `scopes` instead of `scope`, for which this fixed it. Seemed better to start with not cutting corners, since this should have gone in #69 when the rename was made.